cpp/include/raft/sparse/op/detail/filter.cuh

```diff
@@ -46,27 +46,27 @@ template <int TPB_X, typename T>
 RAFT_KERNEL coo_remove_scalar_kernel(const int* rows,
                                      const int* cols,
                                      const T* vals,
-                                     int nnz,
-                                     int* crows,
-                                     int* ccols,
-                                     T* cvals,
-                                     int* ex_scan,
-                                     int* cur_ex_scan,
+                                     uint64_t nnz,
```

Comment:
please add `nnz_t` to template and use that type for `ex_scan` and `cur_ex_scan`.